app/routes/content.js

```diff
@@ -100,12 +100,21 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 ['og:image:width', image.width.toString()],
                 ['og:image:height', image.height.toString()],
                 ['og:image:type', image.type],
-                ['og:url', window.location.href],
+                ['og:url', preprint.get('links.html')],
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated')).toISOString()],
-                ['article:modified_time', new Date(preprint.get('dateModified')).toISOString()]
+                ['article:published_time', dateCreated.toISOString()],
+                ['article:modified_time', new Date(preprint.get('dateModified')).toISOString()],
+                ['citation_title', node.get('title')],
```

Comment:
Can't mix order of tags. The order matters for the scrapers. The OGP tags must be sequential. Updating this in another PR with the redirect fix.